lib/ClangImporter/ImportName.cpp

```diff
@@ -1797,6 +1797,22 @@ ImportedName NameImporter::importNameImpl(const clang::NamedDecl *D,
       isFunction = true;
       addEmptyArgNamesForClangFunction(functionDecl, argumentNames);
       break;
+    case clang::OverloadedOperatorKind::OO_Subscript: {
+      auto returnType = functionDecl->getReturnType();
+      if (!returnType->isReferenceType()) {
+        return ImportedName();
+      }
+      if (returnType->getPointeeType().isConstQualified()) {
+        baseName = "_operatorSubscriptConst";
```

Comment:
I was just thinking... I think that `_operatorSubscript` is actually a valid function name in C++. Only `_Capital` and `__lower` are reserved for the compiler's internal use. So, maybe update this to be double underscore: `__operatorSubscript` so there's no chance we conflict with user code.

Reply:
Oh that's a good point! Added one more underscore to the name.